### DISCUSSION TITLE
Can anyone tell me what this means.

### DISCUSSION THREAD

**Discussion by xedensleavesx** [jef/streetmerchant#1616] [Help]
I have an issue, I only pick up amazon and best buy although i have set the stores to amazon-de and etc, could it be because I get these errors before i run ?

npm WARN hooman@1.2.6 requires a peer of got@>=11.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN twilio@3.54.1 requires a peer of @types/express@^4.17.7 but none is installed. You must install peer dependencies yourself.
npm WARN twilio@3.54.1 requires a peer of @types/qs@6.9.4 but none is installed. You must install peer dependencies yourself.

**Comment by xkuza** [comment]
you are running on castrated linux without proper dependencies. 

Run commands 
npm install --save got@latest
npm install --save  types/express@^4.17.7
npm install --save  types/qs@6.9.4

you possibly will get errors that there is no such thing.. 
And other dependencies missing later.

I downloaded them, and still got into unsandboxed chrome :) in linode.com

got off that and run this occasionaly from my personal machine.
